roach-data-jpa/src/main/java/io/roach/data/jpa/RetryableTransactionAspect.java

```diff
@@ -22,7 +22,7 @@
 
 @Component
 @Aspect
-@Order(Ordered.LOWEST_PRECEDENCE - 2)
+@Order(Ordered.LOWEST_PRECEDENCE - 1)
```

Comment:
Changing the ordering doesn't change the injection order (and logic) as long as there's only one extra advice, so suppose this is ok. It was intentionally similar to the jdbc tutorial for consistency.

Reply:
I see! That makes sense. I suppose it could be really any offset as long as it's a lower number. TFTR!